cpp/memilio/epidemiology/lct_infection_state.h

```diff
@@ -78,6 +81,53 @@ class LctInfectionState
         return index;
     }
 
+    /**
+     * @brief Cumulates a vector with the number of individuals in each subcompartment (with subcompartments 
+     *  according to the LctInfectionState) to produce a Vector that divides the population only into the infection 
+     *  states defined in InfectionStates.
+     *
+     * @param[in] subcompartments Vector with number of individuals in each subcompartment. 
+     *  The size of the vector has to match the LctInfectionState.
+     * @return Vector with accumulated values for the InfectionStates.
+     */
+    static Vector<ScalarType> calculate_compartments(const Vector<ScalarType>& subcompartments)
+    {
+        assert(subcompartments.rows() == Count);
```

Comment:
```suggestion
        assert(subcompartments.rows() == InfectionState::Count);
```

Reply:
No, actually, we want to calculate a vector with InfectionState::Count entries using a vector (subcompartments) with LctInfectionState::Count (which is just Count) entries. (As described in the docu.) Therefore, just Count is right here.